src/redux/reservations/reservations.js

```diff
@@ -0,0 +1,88 @@
+import Axios from 'axios';
+
+const BASE_URL = '';
+
+const FETCH_RESERVATIONS = 'BOOK-APPOINTMENT/RESERVATIONS/FETCH_RESERVATIONS';
+const CREATE_RESERVATION = 'BOOK-APPOINTMENT/MOTORCYCLES/CREATE_RESERVATION';
+const CANCEL_RESERVATION = 'BOOK-APPOINTMENT/MOTORCYCLES/CANCEL_RESERVATION';
```

Comment:
Please add another action_type for fetching a single motorcycle. Example
const FETCH_SINGLE_MOTORCYCLE = 'BOOK-APPOINTMENT/MOTORCYCLES/FETCH_SINGLE_MOTORCYCLE';